pootle/apps/pootle_translationproject/migrations/0004_rename_templates.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from pootle_translationproject.utils import TPTool
+
+
+def rename_templates(apps, schema_editor):
+    projects = apps.get_model("pootle_project.Project").objects.all()
+    TP = apps.get_model("pootle_translationproject.TranslationProject")
+    Language = apps.get_model("pootle_language.Language")
+    to_update = None
+    for project in projects:
+        source_lang = project.source_language
```

Comment:
I would postpone this until we are sure no `templates` TP exists for the project.